ci: remove gulp from the project

romanrostislavovich opened the pull request:
Removed `gulp` and all his dependencies from the project and command for run linting moved to script in `package.json`

Closes #113

xferra commented:
Please doublecheck runtime deps, seems like some of them are dev deps.

romanrostislavovich commented:
`jsonschema` - usage in `configValidation` 
`eslint-config-google` and `eslint` - needs for linting

Other removed packages were used in `gulp`

romanrostislavovich commented:
Sorry, I understood your comment wrong. I fixed it

xferra commented on package.json:
```diff
@@ -10,21 +10,21 @@
-        "chai": "^4.1.2",
-        "commander": "^2.16.0",
-        "mocha": "^5.2.0"
+        "fs": "0.0.1-security",
```
I suppose this is not mandatory.

romanrostislavovich replied:
I fix it

romanrostislavovich committed:
ci: remove gulp from the project

Removed `gulp` and all dependencies from the project and command for run linting moved to script in `package.json`

Closes #113

romanrostislavovich committed:
Merge branch 'develop' into ci/gulp

romanrostislavovich committed:
Merge branch 'develop' into ci/gulp

romanrostislavovich commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@linterhub/usage-parser)
- [GitHub release](https://github.com/linterhub/usage-parser/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
